src/OV/System.cpp

```diff
@@ -3,10 +3,33 @@
 
 #include "System.hpp"
 #include "system/Process.hpp"
+#include "system/FileUtil.hpp"
 
 #include <unistd.h>
 #include <sys/stat.h>
 
+uint_least8_t
+OpenvarioGetBrightness()
+{
+  char line[4];
+  int result = 10;
+
+  if (File::ReadString(Path("/sys/class/backlight/lcd/brightness"), line, sizeof(line))) {
+    result = atoi(line);
+  }
+
+  return result;
+}
+
+void
+OpenvarioSetBrightness(uint_least8_t value)
+{
+  if (value < 1) { value = 1; }
+  if (value > 10) { value = 10; }
+
+  File::WriteExisting(Path("/sys/class/backlight/lcd/brightness"), std::to_string(value).c_str());
```

Comment:
No `std::to_string()` please - `fmt::format_int` is much faster and does not need to allocate anything.

Reply:
Agreed, forgot about our new library.